src/pyaro_readers/nilupmfebas/ebas_nasa_ames.py

```diff
@@ -0,0 +1,806 @@
+    def get_dt_meas(self, np_freq="s"):
+        """Get array with time between individual measurements
+
+        This is computed based on start and timestamps, e.g.
+        `dt[0] = start[1] - start[0]`
+
+        Parameters
+        ----------
+        np_freq : str
+            string specifying output frequency of gap values
+
+        Returns
+        -------
+        ndarray
+            array with time-differences as floating point number in specified
+            input resolution
+        """
+        if self.start_meas is None:
+            self.compute_time_stamps()  # assigns start / stop attrs.
+        start = self.start_meas
+        dts = (start[1:] - start[:-1]).astype(f"timedelta64[{np_freq}]").astype(float)
+        return dts
+
+    def _quality_check(self):
+        msgs = ""
+        if not len(self.data_header) == len(self.var_defs):
+            msgs += (
+                "Mismatch between variable definitions in header and "
+                "number of data columns in table\n"
+            )
+        if not "timezone" in self.meta:
+            msgs += "Timezone not defined in metadata"
+        if msgs:
+            raise AttributeError(f"Quality check failed. Messages: {msgs}")
+
+    def read_header(self, nasa_ames_file, quality_check=True):
+        self.read_file(nasa_ames_file, only_head=True, quality_check=quality_check)
+
+    def read_file(
+        self,
+        nasa_ames_file,
+        only_head=False,
+        replace_invalid_nan=True,
+        convert_timestamps=True,
+        evaluate_flags=False,
+        quality_check=False,
+    ):
+        """Read NASA Ames file
+
+        Parameters
+        ----------
+        nasa_ames_file : str
+            EBAS NASA Ames file
+        only_head : bool
+            read only file header
+        replace_invalid_nan : bool
+            replace all invalid values in the table by NaNs. The invalid values for
+            each dependent data column are identified based on the information in
+            the file header.
+        convert_timestamps : bool
+            compute array of numpy datetime64 timestamps from numeric timestamps
+            in data
+        evaluate_flags : bool
+            if True, all data columns get assigned their corresponding flag
+            column, the flags in all flag columns are decoded from floating
+            point representation to 3 integers, e.g. 0.111222333 -> 111 222 333
+            and if input ```replace_invalid_nan==True```, then the invalid
+            measurements in each column are replaced with NaN's.
+        quality_check : bool
+            perform quality check after import (for details see
+            :func:`_quality_check`)
+        """
+        logger.info(f"Reading NASA Ames file:\n{nasa_ames_file}")
+        lc = 0  # line counter
+        dc = 0  # data block line counter
+        mc = 0  # meta block counter
+        END_VAR_DEF = np.nan  # will be set (info stored in header)
+        IN_DATA = False
```

Comment:
Should be lower case since they aren't constants. https://peps.python.org/pep-0008/#constants

Reply:
adjusted